types/validator/index.d.ts

```diff
@@ -177,7 +177,7 @@ declare namespace ValidatorJS {
     isMultibyte(str: string): boolean;
 
     // check if the string contains only numbers.
-    isNumeric(str: string): boolean;
+    isNumeric(str: string, options?: { no_symbols?: boolean }): boolean;
```

Comment:
I would suggest introducing an interface `IsNumericOptions` in order to adhere to the current structure.